src/facilities-context/FacilitiesContext.tsx

```diff
@@ -44,6 +44,10 @@ export type ExportedActions = {
   duplicateFacility: (facility: Facility) => Promise<Facility | void>;
   deselectFacility: () => void;
   selectFacility: (facilityId: Facility["id"]) => void;
+  fetchReferenceFacilities: (
+    stateName: string,
+    systemType: string,
```

Comment:
Should I update these type to:

```javascript
stateName: Facility["modelInputs"]["stateName"],
systemType: Facility["systemType"],
```

Reply:
personally I don't think that's necessary ... presumably we could change the underlying shape of the facility object without requiring a change to the call signature, as long as we could map from one to the other